packages/dev/docs/pages/blog/building-a-numberfield.mdx

```diff
@@ -26,44 +26,72 @@ import {Flex} from '@react-spectrum/layout';
 
 ---
 keywords: [react aria, react spectrum, react, spectrum, interactions, numberfield, touch, spinbutton]
-description: Numberfields seem like simple components at first, but they hide a lot of complexity under the hood.
-date: 2021-04-08
+description: Number fields are commonly used form components, but are frequently not a great user experience. They often lack support for advanced formatting, such as currency and unit values, and do not provide a localized experience for users around the world. In this post, we'll discuss how we approached building our number field component with support for formatting and internationalization in mind.
+date: 2021-05-05
 author: '[Rob Snow](https://twitter.com/snowystinger)'
 image: ../assets/ReactAria_976x445_2x.png
 ---
-# NumberField
 
-Number fields are often used in web, but are frequently not a great experience.
-In this blog, we'll cover what defines a number field, what issues we ran into with native number fields, and what we've done to improve those issues.
-We'll also go into how number fields can be improved to support internationalization.
-So what is a NumberField.
+# How we internationalized our number field
+
+Number fields are commonly used form components, but are frequently not a great user experience. They often lack support for advanced formatting, such as currency and unit values, and do not provide a localized experience for users around the world. In this post, we'll discuss how we approached building our number field component with support for formatting and internationalization in mind.
 
-## What is a number field
+## What is a number field?
 
 A number field allows users to input and edit numeric values. For example, quantities, dimensions, currencies, percentages, or unit values may be edited with a number field. In addition to allowing the user to enter these values with their keyboard, number fields also often support incrementing and decrementing the value using stepper buttons, the arrow keys, or by scrolling with the mouse.
 
 Values such as credit card and phone numbers are not typically represented by a number field because while they do contain numbers, they are not incrementable, and have additional formatting and validation requirements.
 
-<Image src={numberfieldExample} alt="Example numberfield with formatting" />
+```tsx snippet
+<NumberField label="Width" defaultValue={1024} minValue={0}  />
+```
```

Comment:
Added some live examples rather than the screen shot. Hopefully they work – seems to locally...

Reply:
snippet seems to be including the jsx, did you want that?